src/Dsdevbe/LdapConnector/Adapter/Adldap.php

```diff
@@ -56,6 +56,10 @@ public function getUserInfo($username, $password = null)
     {
         $user = $this->_ldap->search()->where('samaccountname', '=', $username)->first();
 
+        if (!$user) {
+            return;
```

Comment:
Could you also update the DocBlock to reflect the corrent @return null|UserModel